### ISSUE TITLE
Runes not showing in Chrome running in Windows

### ISSUE THREAD

**Issue by martin-henz** [source-academy/cadet-frontend#312]
Here is the report by student Joel:

(1) Your REPL area may not be big enough for the rune. Try making the area bigger, using the slider in the middle or by making the whole window bigger, or use Chome -> View -> Zoom Out.
(2) Your version of Chrome is not recent. Upgrade your Chrome browser to Version 68.

I have tried suggestion (1) to no avail and I am using the latest version of Chrome.

I have tried other means too. All to no avail too.
(1) Settings --> Advanced --> Content Settings --> Images --> Show All.
(2) Disable uBlock Origin (which is the only extension I have on Chrome).
(3) Reinstall Chrome. Hence, default settings.

I installed Firefox and it works! However, I wish to do my Missions on Chrome since it is my default browser. Hence, I hope to assist to resolve the issue for Chrome users.

Attached is the screenshot when I right click and inspect element. (Hopefully it helps!) I am able to right click and save image. Attached is the Saved Image.

Thank you very much!

Warmest Regards,
Joel
![screenshot](https://user-images.githubusercontent.com/30220768/44297386-240f1e00-a303-11e8-9b67-424e5f755e8c.png)

**Comment by JoelLoong** [comment]
I'm using Windows 10 Pro Version 1803.

-Joel

**Comment by ningyuansg** [comment]
@JoelLoong Thank you for the report. Can you help us check if you're getting the warning messages about service-worker in firefox as well?

**Comment by JoelLoong** [comment]
@ningyuansg Nope I'm not. Everything seems to be working fine in Firefox.

**Comment by ningyuansg** [comment]
1. Could you expand the warning messages in Google Chrome? Clicking on the carets (triangles) on the left sides of the messages should do this. I'm looking to see if they can tell us which 'cached response' in particular is missing.

2. Could you clear your cache? Reinstalling chrome may not clear the cache since some uninstall processes don't bother to remove *all* their files.

**Comment by JoelLoong** [comment]
![screenshot](https://user-images.githubusercontent.com/42489904/44307761-d534b780-a3da-11e8-8dde-7b896ef755d9.PNG)

1. Attached.

2. Forgot to mention, I have already cleared the cache before submitting this request.

**Comment by ningyuansg** [comment]
Thanks! We'll investigate this further, but I must admit that pushing a fix before this series of rune missions are over is unlikely. You may have to depend on firefox for this month's source academy assessments. Again, sorry about that.

**Comment by JoelLoong** [comment]
As suggested by Jason Chee, I'm now able to view the images after enabling **Use Hardware Acceleration When Available**.

**Comment by ReignOfComputer** [comment]
Right, forgot to post here too (Jason here). I looked into detection of HW Acceleration but there's no good way to do it, and some devices will have software rendering that works. Seems to be a problem with Chrome's rendering, may want to file a bug report with Chromium/ANGLE.

CP from Piazza:
"Hi, got my hands on my device that doesn't display the runes, in Chrome. I fixed it by going to Chrome Settings > Advanced > System > Use hardware acceleration when available, then relaunch Chrome. Hope this helps."
